src/todolist/CommandParserImpl.java

```diff
@@ -0,0 +1,25 @@
+package todolist;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+public class CommandParserImpl implements CommandParser {
+
+    @Override
+    public ParseResult parse(String source) {
+        String[] sourses = source.trim().split(" ");
+        String commandStr = sourses[0].toUpperCase(Locale.ROOT);
+        try {
+            Command command = Command.valueOf(commandStr);
+            String[] parameters = Arrays.copyOfRange(sourses, 1, sourses.length);
+            ParseResult.Success success = new ParseResult.Success(command, parameters);
```

Comment:
лишняя переменная. Я советую от таких переменных лучше избавляться, код становиться чище. На любителя и по соглашению может быть приемлемо. Но работая вдвоем, давай договоримся не оставлять такие переменные.
Я не буду указывать их везде, сама пройдись по коду и посмотри.
```suggestion
            return new ParseResult.Success(command, parameters);
```